ci/jenkins/test-vmc.sh

```diff
@@ -439,10 +449,12 @@ function deliver_antrea {
     do
         ssh-keygen -f "/var/lib/jenkins/.ssh/known_hosts" -R ${IPs[$i]}
         if [[ "$COVERAGE" == true ]]; then
-            copy_image antrea-ubuntu-coverage.tar docker.io/antrea/antrea-ubuntu-coverage ${IPs[$i]} ${DOCKER_IMG_VERSION} true
+            copy_image antrea-ubuntu-coverage.tar docker.io/antrea/antrea-agent-ubuntu-coverage ${IPs[$i]} ${DOCKER_IMG_VERSION} true
+            copy_image "" docker.io/antrea/antrea-controller-ubuntu-coverage ${IPs[$i]} ${DOCKER_IMG_VERSION} false
             copy_image flow-aggregator-coverage.tar docker.io/antrea/flow-aggregator-coverage ${IPs[$i]} ${DOCKER_IMG_VERSION} true
         else
-            copy_image antrea-ubuntu.tar docker.io/antrea/antrea-ubuntu ${IPs[$i]} ${DOCKER_IMG_VERSION} true
+            copy_image antrea-ubuntu.tar docker.io/antrea/antrea-agent-ubuntu ${IPs[$i]} ${DOCKER_IMG_VERSION} true
+            copy_image "" docker.io/antrea/antrea-controller-ubuntu ${IPs[$i]} ${DOCKER_IMG_VERSION} false
```

Comment:
line 424 is still using antrea-ubuntu:latest because it is related to old version of antrea.

Reply:
Yes it feels like we may have to address it in the follow up PR.
I think it will only be broken after we release 1.15, and we start testing upgrade from Antrea v1.15 to a later version. At this point we may have to account for the fact that the old version can either be using a unified image or split images.

Reply:
> I noticed that we also use old Antrea image in this script from line 462. I feel this might be broken after image split. @XinShuYang could you help to estimate? Thanks.

Yes, we only pull old image in `{name}-{test_name}-matrix-compatibility-test`, I am okay to fix it in a separate PR after 1.15 release.